test/webhost/test_option_presets.py

```diff
@@ -14,6 +15,10 @@ def test_option_presets_have_valid_options(self):
                     with self.subTest(game=game_name, preset=preset_name, option=option_name):
                         try:
                             option = world_type.options_dataclass.type_hints[option_name].from_any(option_value)
+                            # some options may need verification to ensure the provided option is actually valid
+                            # pass in all plando options in case a preset wants to require certain plando options
+                            # for some reason
+                            option.verify(world_type, "Test Player", PlandoOptions(0b1111))
```

Comment:
Pretty sure that does the same thing
```suggestion
                            option.verify(world_type, "Test Player", PlandoOptions(sum(PlandoOptions)))
```